test/index.js

```diff
@@ -141,14 +147,12 @@ describe('simple-keyring', function () {
       signatures.forEach((sgn, index) => {
         const accountAddress = addresses[index];
 
-        const r = ethUtil.toBuffer(sgn.slice(0, 66));
-        const s = ethUtil.toBuffer(`0x${sgn.slice(66, 130)}`);
-        const v = ethUtil.bufferToInt(
-          ethUtil.toBuffer(`0x${sgn.slice(130, 132)}`),
-        );
-        const m = ethUtil.toBuffer(msgHashHex);
-        const pub = ethUtil.ecrecover(m, v, r, s);
-        const adr = `0x${ethUtil.pubToAddress(pub).toString('hex')}`;
+        const r = toBuffer(sgn.slice(0, 66));
+        const s = toBuffer(`0x${sgn.slice(66, 130)}`);
+        const v = BigInt(`0x${sgn.slice(130, 132)}`);
```

Comment:
new version of ecrecover expects v to be a BigInt instead of Buffer: https://npmfs.com/package/@ethereumjs/util/8.0.0/src/signature.ts#L54